### PR TITLE
Home Page for Fabrik

### PR DESCRIPTION
**Pull request by sayamkanwar**
Hi,
I have created the homepage for Fabrik. I have added all the deliverables. The team section currently has sample data, I will replace that with the content when the data is available.

Please review @Ram81 @RishabhJain2018 @utsavgarg 
Thank you! :)

### CODE REVIEW

**Comment by sayamkanwar** [comment]
Added content for the team section.
@Ram81

**Comment by sayamkanwar** [comment]
Hi,
I have made the changes.
Please review @deshraj @virajprabhu
Thank you

**Comment by sayamkanwar** [comment]
@Ram81, made the changes!

**Comment by sayamkanwar** [comment]
@Ram81, made all the changes!

Here's a demo

demo_homepage

**Comment by sayamkanwar** [comment]
@Ram81, so no demos for the features?

**Comment by sayamkanwar** [comment]
Is this fine? @utsavgarg @RishabhJain2018 

screenshot 2018-12-04 at 11 58 46 pm

**Comment by sayamkanwar** [comment]
Or would you like something like this?

example

**Comment by PalashTanejaPro** [comment]
If I may add to the discussion, how about GIFs that have a translucent play button on them and play when you hover on them

**Comment by sayamkanwar** [comment]
@PalashTanejaPro Thanks for the suggestion, but I guess that would be a little out of place because these GIFs were being triggered by clicking on a feature box. So even after once the user has clicked the box, a play button is required? I can add it but it might spoil the user experience.

**Comment by sayamkanwar** [comment]
@Ram81 Oh I'm really sorry, I'll fix it.

**Comment by sayamkanwar** [comment]
Is this fine? @Ram81 

screenshot 2018-12-07 at 5 39 40 pm

**Comment by sayamkanwar** [comment]
@Ram81 Changed the design to what you asked for. I hope it's fine now?

**Comment by sayamkanwar** [comment]
Is this what you want? @Ram81 
screenshot 2018-12-08 at 2 31 32 pm

**Comment by sayamkanwar** [comment]
Actually, alignment was still a bit messed up, I have corrected it now. Here:
screenshot 2018-12-08 at 5 39 05 pm

**Comment by sayamkanwar** [comment]
Sure @Ram81

**Comment by sayamkanwar** [comment]
@Ram81, the homepage should be available at /#/home

**Comment by sayamkanwar** [comment]
@Ram81, the homepage would be available at /#/home. :)

File: ide/static/js/home.js [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+                    </div>
+                    <img src={this.state.demo_gif_3} className="demo-gif" />
+                  </div>
+                </div>
+                <div className="demo-right">
+                  <h2 style={{ wordSpacing: "-0.08em" }}>
+                    Build Model From Components
+                  </h2>
+                  <p>
+                    There are a lot of components, called layers, provided which
```
Comment (Ram81) [inline ide/static/js/home.js]: Can you change this to some statement which is more descriptive and clear

File: ide/static/js/home.js [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+                    you can use to create neural nets.
+                  </p>
+                </div>
+              </div>
+              <div className="demo">
+                <div className="demo-right">
+                  <h2>Export Model</h2>
+                  <p>You can export your model in any of the 3 frameworks.</p>
```
Comment (Ram81) [inline ide/static/js/home.js]: Make more descriptive

File: ide/static/js/home.js [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+                        />
+                      </center>
+                    </div>
+                    <img src={this.state.demo_gif_5} className="demo-gif" />
+                  </div>
+                </div>
+                <div className="demo-right">
+                  <h2>Import Model</h2>
+                  <p>
+                    You can import model files from your local system and load
```
Comment (Ram81) [inline ide/static/js/home.js]: Same here

File: ide/static/js/home.js [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+                    them in the canvas.
+                  </p>
+                </div>
+              </div>
+              <div className="demo">
+                <div className="demo-right">
+                  <h2>Share Model</h2>
+                  <p>
+                    You can generate a shareable link for your model which
```
Comment (Ram81) [inline ide/static/js/home.js]: Rephrase this

File: ide/static/js/home.js [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+                        />
+                      </center>
+                    </div>
+                    <img src={this.state.demo_gif_7} className="demo-gif" />
+                  </div>
+                </div>
+                <div className="demo-right">
+                  <h2>Real Time Collaboration</h2>
+                  <p>
+                    You can collaborate with someone else and edit your model in
```
Comment (Ram81) [inline ide/static/js/home.js]: Rephrase

**Comment by sayamkanwar** [comment]
@Ram81, please check now.

**Comment by sayamkanwar** [comment]
Thank you! @RishabhJain2018,
1. Yes I know that I thought I should keep it a bit different but I'll change it no issues.
2. Sure.
3. I'll look into it.
4. So image above text? Or text above the image?

**Comment by sayamkanwar** [comment]
@Ram81 @RishabhJain2018, Made the changes! Please have a look. :)

**Comment by sayamkanwar** [comment]
screenshot 2018-12-09 at 12 33 43 am

**Comment by sayamkanwar** [comment]
@Ram81 It was already that. I think this problem is because of the screen size. I'll fix it.

**Comment by RishabhJain2018** [comment]
Yeah, I apologize for the confusion @sayamkanwar.
It should be left aligned only but it should be horizontally centered with the image. Currently, it is top aligned with the center image.
Please feel free to ask if there are still doubts.

**Comment by sayamkanwar** [comment]
@Ram81 @RishabhJain2018 Done! :)

**Comment by sayamkanwar** [comment]
@Ram81 I had already fixed that in my last commit where @RishabhJain2018 had pointed out! Please have a look at that. :)

**Comment by Ram81** [comment]
@sayamkanwar I tested it just now and it didn't work for me

**Comment by sayamkanwar** [comment]
Fixed @Ram81 

features_scroll

File: ide/static/css/home_style.css [hunk 6]
```diff
@@ -0,0 +1,303 @@
+a {
+  text-decoration: none;
+}
+a:hover {
```
Comment (Ram81) [inline ide/static/css/home_style.css]: Add newline above this

File: ide/static/css/home_style.css [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+  text-decoration: none;
+}
+body {
```
Comment (Ram81) [inline ide/static/css/home_style.css]: Same here

File: ide/static/css/home_style.css [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+	background: #f9fcff;
+}
+.home-container {
```
Comment (Ram81) [inline ide/static/css/home_style.css]: Same here

File: ide/static/css/home_style.css [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+  width: 100%;
+  margin: 0;
+  font-family: 'Roboto',sans-serif;
+  padding: 0;
+  background: #f9fcff;
+  height: 100%;
+}
+.header {
```
Comment (Ram81) [inline ide/static/css/home_style.css]: Same here

File: ide/static/css/home_style.css [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+  float: right;
+  width: 20em;
+}
+.icons {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  margin-top: 3.6em;
+}
+.icons img {
```
Comment (Ram81) [inline ide/static/css/home_style.css]: Add newline above every new css class

**Comment by gautamjajoo** [comment]
Hey! If I may add a point to the discussion (just a suggestion) .This would be a homepage for Fabrik but we don't have the logo of Fabrik anywhere on the page. We could add it on the landing page maybe?

**Comment by sayamkanwar** [comment]
@Ram81, it was left aligned initially but @RishabhJain2018 had asked to align it in the center of the left margin. So should I change it back to what it was?

**Comment by sayamkanwar** [comment]
@gautamjajoo Thanks for the suggestion! Could you please suggest a good place for the Fabrik’s logo to be added as the header already contains CloudCV’s logo?

**Comment by Ram81** [comment]
@sayamkanwar yes please left align the title

**Comment by sayamkanwar** [comment]
Okay sure @Ram81.

**Comment by sayamkanwar** [comment]
1. Fixed webpack errors.
2. Replaced cloudcv's logo with fabrik's logo
3. Left aligned the text

@Ram81

**Comment by Ram81** [comment]
@sayamkanwar also do one thing in features part lets keep three features for now

1. Load from Zoo
2. Export Model
3. Share Model (Share model and RTC are one and the same)

File: ide/static/js/home.js [hunk 11]
```diff
@@ -0,0 +1,505 @@
...
+                        />
+                      </center>
+                    </div>
+                    <img src={this.state.demo_gif_7} className="demo-gif" />
+                  </div>
+                </div>
+                <div className="demo-right">
+                  <h2>Real Time Collaboration</h2>
+                  <p>
+                    You can work jointly with other people,
```
Comment (Ram81) [inline ide/static/js/home.js]: Rephrase to `You can work collaboratively`

**Comment by sayamkanwar** [comment]
Closing this PR. I will open a new PR for the homepage.

**Pull request closed without merging**